ts-npm-example/README.md

```diff
@@ -0,0 +1,38 @@
+# Basic TypeScript and NPM example of SDK usage
+This basic axample shows how simple it is to make a TypeScript integration, as well as using the SDK via NPM instead of a script.
```

Comment:
example

Reply:
Thanks, done in [2443357](https://github.com/chili-publish/editor-sdk-integration-examples/pull/3/commits/2443357083cdad76904eeee88cd487d70c21d320)